arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,410 @@
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "DatabaseAuditingPolicyProperties": {
+            "properties": {
+                "useServerDefault": {
+                    "type": "string",
+                    "description": "Indicates whether the default server auditing policy is used for that Azure SQL Database."
+                },
+                "auditingState": {
+                    "type": "string",
+                    "description": "Gets the auditing state of the Azure SQL Server auditing policy."
+                },
+                "eventTypesToAudit": {
+                    "type": "string",
+                    "description": "Gets the events types of the Azure SQL Server auditing policy."
+                },
+                "storageAccountName": {
+                    "type": "string",
+                    "description": "Gets the storage account name of the Azure SQL Server auditing policy."
+                },
+                "storageAccountKey": {
+                    "type": "string",
+                    "description": "Gets the primary storage account key of the Azure SQL Server auditing policy."
+                },
+                "storageAccountSecondaryKey": {
+                    "type": "string",
+                    "description": "Gets secondary key of the storage account of the Azure SQL Server auditing policy."
+                },
+                "storageTableEndpoint": {
+                    "type": "string",
+                    "description": "Gets the storage table endpoint of the Azure SQL Server auditing policy."
+                },
+                "storageAccountResourceGroupName": {
+                    "type": "string",
+                    "description": "Gets the resource group of the storage account of the Azure SQL Server auditing policy."
+                },
+                "storageAccountSubscriptionId": {
+                    "type": "string",
+                    "description": "Gets the subscription Id of the storage account of the Azure SQL Server auditing policy."
+                },
+                "retentionDays": {
+                    "type": "string",
```

Comment:
Our service expects this to be a string. In future API versions we will probably change this to be the appropriate type.

@yoavrubin Could you comment on what the expected values are and update the documentation.